idaes/models_extra/gas_solid_contactors/unit_models/tests/test_FB0D.py

```diff
@@ -176,21 +178,231 @@ def test_build(self, iron_oc):
     def test_dof(self, iron_oc):
         assert degrees_of_freedom(iron_oc) == 0
 
+    @pytest.fixture(scope="class")
+    def iron_oc_unscaled(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(
+            default={"dynamic": True, "time_set": [0, 3600], "time_units": pyunits.s}
+        )
+
+        m.fs.gas_props = GasPhaseParameterBlock()
+        m.fs.solid_props = SolidPhaseParameterBlock()
+        m.fs.solid_rxns = HeteroReactionParameterBlock(
+            default={
+                "solid_property_package": m.fs.solid_props,
+                "gas_property_package": m.fs.gas_props,
+            }
+        )
+
+        m.fs.unit = FixedBed0D(
+            default={
+                "gas_property_package": m.fs.gas_props,
+                "solid_property_package": m.fs.solid_props,
+                "reaction_package": m.fs.solid_rxns,
+            }
+        )
+
+        # Discretize time domain
+        m.discretizer = TransformationFactory("dae.finite_difference")
+        m.discretizer.apply_to(m, nfe=100, wrt=m.fs.time, scheme="BACKWARD")
+
```

Comment:
I suggest reducing the time_set to 360s and nfe to 10 in the iron_oc_unscaled method. This will resolve the issue with the failing "test_initialize_unscaled" test. Ideally, at the given initial conditions, and time horizon of 3600 you will want to apply a scaling transformation before solving this model.